test/models/newbooking_test.rb

```diff
@@ -3,7 +3,37 @@
 require 'test_helper'
 
 class NewbookingTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'Control' do
+    pet = Pet.all[0]
+    booking = Newbooking.new
```

Comment:
Remembering that this is just ruby... think about DRY.

When I'm doing something like this, I might create a helper factory method that will create an instance of the object with everything set "to work". Then when I want to test a "failure case", I might call it and alter just the one thing I want to test

Reply:
Ok that makes sense.